micro-rdk/src/common/conn/server.rs

```diff
@@ -356,9 +354,18 @@ where
             network,
         }
     }
-    pub async fn serve(&mut self, robot: Arc<Mutex<LocalRobot>>) {
+    pub async fn serve(&mut self, robot: Arc<Mutex<LocalRobot>>, app_client: Option<AppClient>) {
```

Comment:
Very likely so. It was on my list of small improvements to try but I didn't pursue it in the interest of getting this out for review. I'll try it and if it works I'll make the change. I expect it will.